### ISSUE TITLE
rctmon_energy_household_sum is twice as much as Self-consum in the app. 

### ISSUE THREAD

**Issue by poggenpower** [svalouch/python-rctclient#24]
If I compare the difference of rctmon_energy_household_sum for a timeframe with the self-consume report from the App,
rctmon_energy_household_sum has counted roughly twice as much kWh as what the app is showing if I create a history report for "Self-consum" in the app.

Analyzing the pcap from a history report generation in the App, it is running a query for 2 timeseries

- logger.XXXX_eac_log_ts
- logger.XXXX_egrid_feed_log_ts

I found out that the Self-Consum report in the App is calculated as:
logger.XXXX_eac_log_ts + logger.XXXX_egrid_feed_log_ts

Means there is no real logger that stores the self-consum. As egrid_feed is kind of self explanatory, I would like to know what `eac`  means. 

As a further item Device > Household XXXX energy correlates to rctmon_energy_household_sum.

Long story short. I don't think there is an issue with the rct-client itself, but I would like to know what `eac` stands for and if you see the same pattern in your environment.

**Comment by poggenpower** [comment]
I have answered the question for "eac" by myself: RTFM
<img width="914" alt="image" src="https://user-images.githubusercontent.com/6035034/229936184-80b80855-93fa-426e-9050-ffa3a68dc45a.png">
eac: Inverter output energy

And now I have also answered the 2nd part of my question. For the household it adds also the energy loaded from the grid. 
Thanks sometimes it helps to write down all the things ;-)
